### ISSUE TITLE
Inprovements of the beta-version 0.13.99

### ISSUE THREAD

**Issue by Tradefreak** [namecoin/namecoin-core#305]
Probably some of the following issues have/are been discussed already, but I summarize some problems with the current node, installed over the namecoin-0.13.99-win64-setup-unsigned.exe, Qt version 5.6.1, connected over a 30 Mbit LTE-connection:

1. For new name operations the name should be shown internal at the transactions tab.
2. Accidentally repeated new name operations should generate a popup with "already been registered", else it will be operated twice or more.
3. All the time a "insufficient funds"-problem occurs, mostly after ~ 10 directly consecutive operations (new name or name updates/first updates). Sometimes the correct height of funds is shown, but the popup appears: _"Error: The transaction was rejected! This might happen if some of the coins in your wallet were already spent, such as if you used a copy of the wallet and coins were spent in the copy but not marked as spent here."_
4. When the "insufficient funds"-problem occurs after a new name operation, there's no possibility anymore to type in the data of the value doe to the "insufficient funds"-popup. So the name operation is payed, but useless, the popup for typing in the data of the value never appears. This happens even several times for the same operation.
5. After restarting the application a hundred times for a reindexing of the blocks after a "insufficient funds"-problem (which makes only sense if the last transactions have at least one confirmation), several records of the same name appear twice in the registered name tab, one time with the correct value of remaining blocks, another one with an empty field of remaining blocks. The problem disappears after another restart of the application.
6. Name operations should be possible to be added as a list in any length, so that not every transaction has to be confirmed with password and value separately.
7. If the encrypted wallet is unlocked one time by the password, the password shouldn't be requested a hundred times again for the following transactions.
8. Unnecessary second confirmation of the name's first update, which has to be canceled, at least it should have the option to be confirmed automatically. Unclear for a new user, whether clicking "ok" results in a lost of the name or not and what to do.
9. Insistent and unnecessary "already been registered" popup for name first updates, at least it should have the option to be confirmed automatically.
10. When clicking away like a robot a hundred times the second name confirmation or the "already been registered" popup, one accidentally clicks at "ok" of a suddenly at the same location appearing "insufficient funds"-popup, resulting in a cancellation of the transaction. Unclear, if the name is lost or what to do.
11. Accidentally added wrong domains or no longer needed domains can't be deleted from the list of names. Therefor should exist at least a separate tab for expired domains.
12. Missing possibility of an export of all registered names with values.
13. Missing a search field in the registered names tab.
14. Missing acoustical notice and/or desktop notification for confirming name registration popups after 12 blocks.
15. Missing a function of automated name updates under a specific threshold (e.g. at 20 blocks remaining).

**Comment by somewhatnmc** [comment]
This seems to be quite heavy on the Qt side of things so I will bandwagon with: https://github.com/namecoin/proposals/blob/master/ifa-0001.md >>>>> https://wiki.namecoin.info/index.php?title=Domain_Name_Specification(currently Error code: SEC_ERROR_EXPIRED_CERTIFICATE) for the Manage Names tab's "Domain Names" link. Also for point 6/7 name_autoregister is in electrum-nmc so could be added to core?

**Comment by JeremyRand** [comment]
Please file 1 issue per bug/request.  It's not feasible to track all of these things in 1 issue.

**Comment by Tradefreak** [comment]
For sure I won't open 15 cases, especially not as the facts listed above are dependent from each other! Discuss and resolve it or let it be as it is - it doesn't matter for me!

**Comment by JeremyRand** [comment]
> Therefor I numbered each separate point for easy discussion! For sure I won't open 15 cases, especially not as the facts listed above are dependent from each other! Discuss and resolve it or let it be as it is - it doesn't matter for me!

@Tradefreak You don't seem to understand how issue trackers work.  We should not be expected to derail the standard issue tracker workflow just because you're too lazy to follow established procedures.

**Comment by Tradefreak** [comment]
EOD from my side!

**Comment by somewhatnmc** [comment]
@Tradefreak I think more detail is needed on a few of your suggestions. I would say add a comment on the issues created above/being created and ELI5 what steps you're taking and what is occurring from core so that the behavior you want improved can be understood. Otherwise there may not be enough information without your input and they'll be closed off.

**Comment by Tradefreak** [comment]
@somewhatnmc Thanks for your great assistance! I really didn't want to spam the project here with all separate issues, and as you mentioned it correctly in the separately opened issues, most of them are related to each other. If one ore more specific points are unclear, please let me know! You meant the remaining points you still didn't reopen separate? BTW, I have more issues noted offline, which I will open here later. Thx so far!

**Comment by Tradefreak** [comment]
@domob1812 Darf ich fragen, wo die ganzen Threads hin sind, die Du da aus diesem hier generiert hast?
